client/src/utils/convertDate.js

```diff
@@ -0,0 +1,27 @@
+import * as Constants from './constants'
```

Comment:
I see that you've been getting a lot of practice with converting stuff, but at this point, for practicality's sake, consider using a library like `moment.js`